src/GraphQL.Tests/Types/AutoRegisteringInterfaceGraphTypeTests.cs

```diff
@@ -335,35 +336,23 @@ public void CanOverrideFieldList()
     }
 
     [Theory]
-    [InlineData("Field1", 1)]
-    [InlineData("Field2", 2)]
-    [InlineData("Field4", 4)]
-    [InlineData("Field6AltName", 6)]
-    [InlineData("Field7", 7)]
-    public async Task FieldResolversWork(string fieldName, object expected)
+    [InlineData("Field1")]
+    [InlineData("Field2")]
+    [InlineData("Field4")]
+    [InlineData("Field6AltName")]
+    [InlineData("Field7")]
+    public async Task FieldResolvers_Invalid(string fieldName)
     {
         var graph = new AutoRegisteringInterfaceGraphType<TestInterface>();
         var field = graph.Fields.Find(fieldName).ShouldNotBeNull();
         var resolver = field.Resolver.ShouldNotBeNull();
         var obj = new TestClass();
-        var actual = await resolver.ResolveAsync(new ResolveFieldContext
+        var ex = await Should.ThrowAsync<InvalidOperationException>(async () => await resolver.ResolveAsync(new ResolveFieldContext
         {
             Source = obj,
             FieldDefinition = new FieldType { Name = fieldName },
-        }).ConfigureAwait(false);
-        actual.ShouldBe(expected);
-    }
-
-    [Fact]
-    public async Task WorksWithNoDefaultConstructor()
```

Comment:
This removed test doesn't make any sense for an interface -- they never have a default constructor.